app/models/manageiq/providers/openstack/refresh_parser_common/helper_methods.rb

```diff
@@ -3,15 +3,15 @@ module Openstack
     module RefreshParserCommon
       module HelperMethods
         def openstack_admin?
-          ::Settings.ems.ems_refresh.openstack.try(:is_admin)
+          ::Settings.ems_refresh.openstack.try(:is_admin)
```

Comment:
a nit for other PR probably:

we usually abstract ```::Settings.ems_refresh.openstack``` as a method , in new refresh parser it available on collector https://github.com/Ladas/manageiq/blob/b9e6b294604d744f59e3bf270e16264f067fb85a/app/models/manager_refresh/inventory/collector.rb#L11

Reply:
I agree that it should be abstracted out in the future.